src/Stratis.Bitcoin/Connection/PeerBanning.cs

```diff
@@ -128,11 +81,16 @@ public void BanPeer(IPEndPoint endpoint, int banTimeSeconds, string reason = nul
                 }
             }
 
-            if (banPeer)
-            {
-                this.logger.LogDebug("Peer '{0}' banned for reason '{1}'.", endpoint, reason);
-                this.banStore.BanPeer(endpoint, this.dateTimeProvider.GetUtcNow().AddSeconds(banTimeSeconds));
-            }
+            if (!banPeer) return;
```

Comment:
Please put log traces inside this `if`s

Reply:
Sorry I don't follow.

If banPeer is false (line 84) then return, as no need to go to AddressManager to get the ban details.  Otherwise carry on.  

I have added a LogDebug at line 93 - to log BanDate and BanUntil values (if the Peer is found).  Should I change this to LogTrace ?

Reply:
I will explain on the call